### PR TITLE
Add cryptocurrency address queries / insights

### PR DESCRIPTION
**Pull request by morriscode**
Adding insights of crypto currency wallets:
bech32
bitcoin
bitcoin cash
dash
ethereum
litecoin
monero

### CODE REVIEW

File: queries/content/bech32_address.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: "bech32_address"
```
Comment (jkamdjou) [inline queries/content/bech32_address.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Bech32"
```

File: queries/content/bitcoin_address.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+name: "bitcoin_address"
```
Comment (jkamdjou) [inline queries/content/bitcoin_address.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Bitcoin"
```

File: queries/content/bitcoin_cash.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+name: "bitcoin_cash"
```
Comment (jkamdjou) [inline queries/content/bitcoin_cash.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Bitcoin Cash"
```

File: queries/content/dash_wallet.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+name: "dash_wallet"
```
Comment (jkamdjou) [inline queries/content/dash_wallet.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Dash"
```

File: queries/content/ethereum_address.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+name: "ethereum_address"
```
Comment (jkamdjou) [inline queries/content/ethereum_address.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Ethereum"
```

File: queries/content/litecoin_address.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+name: "litecoin_address"
```
Comment (jkamdjou) [inline queries/content/litecoin_address.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Litecoin"
```

File: queries/content/monero_address.yml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+name: "monero_address"
```
Comment (jkamdjou) [inline queries/content/monero_address.yml:1]:
```suggestion
name: "Cryptocurrency wallet address: Monero"
```

**Review by jkamdjou — COMMENTED** [review]
🔥 awesome adds. small suggestion for the names to make them a little more user friendly + consistent with the other Insights that show up in the Dashboard. i'm not certain "wallet address" is technically correct for all of these, so feel free to rev on that however you see fit

great work!

File: queries/content/bech32_address.yml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+name: "Cryptocurrency address: Bech32"
+type: "query"
+source: |
+  any([body.plain.raw, body.html.raw], regex.match(., '\b(bc(0([ac-hj-np-z02-9]{39}|[ac-hj-np-z02-9]{59})|1[ac-hj-np-z02-9]{8,87}))\b'))
```
Comment (rw-access) [inline queries/content/bech32_address.yml:4]: WDYT about switching these to `body.html.display_text` or `body.html.inner_text`? raw HTML can be pretty large to sift through

Comment (morriscode) [reply queries/content/bech32_address.yml:4]: Great idea, will update these to use body.html.display_text

Comment (morriscode) [reply queries/content/bech32_address.yml]: @rw-access updated

**Review by rw-access — APPROVED** [review]

**Pull request merged**